api/src/test/java/org/openmrs/ui/framework/page/PageContextTest.java

```diff
@@ -61,4 +62,17 @@ public void getUniqueResourcesByCategory_shouldGetCorrectOrder() throws Exceptio
 		Collection<Resource> resources = pageContext.uniqueSortedResourcesByCategory(Resource.CATEGORY_JS);
 		assertThat(resources, contains(third, first, second));
 	}
+	
+	@Test
+	public void includeResource_shouldNotIncludeDuplicateResources() {
+		Resource first = new Resource(Resource.CATEGORY_JS, "source", "z_first", 1);
+		Resource second = new Resource(Resource.CATEGORY_JS, "mergepatientdata", "path", 2);
```

Comment:
It shows using this Constructor, the 'second' is of higher priority to 'duplicateResource' so it takes precedence @dkayiwa  though gonna add code to prove that